harness/determined/common/declarative_argparse.py

```diff
@@ -242,3 +243,8 @@ def description_sort_key(desc: Any) -> str:
     # the default print help.
     if subparsers is not None and parser.get_default("func") is None:
         parser.set_defaults(func=help_func(parser))
+
+
+# Converts string values to boolean for flag arguments (e.g. --active=true)
```

Comment:
We're not the best at [following our style guide](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods), here, (especially in the CLI), but could you please make this a one-line docstring?